index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href = "style.css">
+        <title>Tamashevich CV</title>
+        </title>
+    </head>
+    <body>
+        <div>
+            <h1>Sviatlana Tamashevich</h1>
+        </div>
+        <div>
+            <h2>Junior software developer</h2>
+        </div>
+        <img src = "photo.jpg" alt = "photo" />
+        <div>
+            <h3>SUMMARY</h3>
+            <p>A graduated physicist and experienced engineer who works with different types of devices
+                such as medicine analyzers, spectrophotometers, refractometers etc. Responsible, logical
+                thinking, hardworking and able to make decisions. Have a passion for a career in developing.
+                Good motivated and keen to learn.</p>
+            <p>Currently looking for a software developer opportunity to change carrier and have a new work
+                experience alongside an expert team of developers. Thereby helping to achieve required
+                skills for developing carrier and make best use of my existing skills.</p>
+        </div>
+        <div>
+            <h3>QUALIFICATIONS</h3>
+            <p>Master's degree in physics<br>
+               Belorussian State University 2002 - 2007
+            </p>
+            <p>Foreign language:<br>
+               English, A2
+            </p>
+            <p><strong>Courses and seminars</strong>
+                <ol>
+                    <li>On-line course: Codeacademy.com</li>
+                        <ul>
+                            <li>Introduction to HTML</li>
+                            <li>Learn CSS</li>
+                        </ul>
+                    <li>RSSchool lectures on working:</li>
+                        <ul>
+                            <li>Git</li>
+                            <li>Basics of JS</li>
+                        </ul>
+                    <li>Vitaliy Fridman's workshop “New Adventures in Front-End”   September 2018</li>
```

Comment:
```
<li>RSSchool lectures on working:</li>
  <ul>
      <li>Git</li>
      <li>Basics of JS</li>
  </ul>
<li>Vitaliy Fridman's workshop “New Adventures in Front-End”   September 2018</li>
```
would be better something like this, where `<ol>/<ul>` have only  `li` as first level child
```
<li>
  <p>RSSchool lectures on working:</p>
  <ul>
      <li>Git</li>
      <li>Basics of JS</li>
  </ul>
</li>
<li>Vitaliy Fridman's workshop “New Adventures in Front-End”   September 2018</li>
```